presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -558,4 +562,60 @@ else if (value instanceof Document) {
 
         return Optional.ofNullable(typeSignature);
     }
+
+    private String getDatabaseName(String lname)
```

Comment:
We don't use an abbreviation. Let's rename `lname` and the following `a`, `lDatabaseName` and `lTableName`